React-native.md

```diff
@@ -0,0 +1,84 @@
+# React-Native 
+
+React Native is a JavaScript framework Allow you to writes real, natively rendering mobile applications for iOS and Android ,based on React.
+You can create code just once and use it to power both their iOS and Android apps. This translates to huge time and resource savings. ,React Native has the ability to render APIs in Objective-C (for iOS) or Java (for Android),your application will render using “bridge” to become real mobile UI components,and will look and feel like any other mobile application,your React Native apps can access platform features like the phone camera, or the user’s location.
```

Comment:
Can you talk more about the bridge? What is it? How does it work?

Once you have clarified what the bridge is, you should be able to clearly communicate the relationship between JavaScript and native components (Java on Android, Objective C on iOS).